test/test-helpers/mocked-context.ts

```diff
@@ -1,11 +1,27 @@
 import { Context } from "cloudworker-router";
+// This is to make Request and other browser stuff work
+import "isomorphic-fetch";
```

Comment:
but Node 18 has fetch.  Do you not have NVM running automatically?

Reply:
Had to add it to get the typing for the Request object to work.
Then there's another annoying issue that the github-ci action for wrangler doesn't work with node 18..